tokens/utils.js

```diff
@@ -162,7 +162,37 @@ async function transformInPath(location, variablesMap, transformType = 'definiti
   }
 }
 
+
+function createIndexFile(buildDir, isTheme, themeVariant){
```

Comment:
nit: tiny naming nit; perhaps something like `createIndexCssFile` just to be a tad bit more explicit in what type of file is being created by this function?